app/players/elves/elf.py

```diff
@@ -0,0 +1,13 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
The `Elf` class should be abstract as well because it does not implement all abstract methods from the `Player` class.